Chart.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
 name: vault
```

Comment:
```suggestion
name: vault
type: application
```

Reply:
Thanks for the review! I think we'll leave this out for now since it looks like `application` is the default.

Reply:
🙏